src/main/webapp/view/transcript/transcript.js

```diff
@@ -103,39 +99,71 @@ class TranscriptLineElement extends HTMLElement {
   /**
    * Creates a custom HTML element representing `transcriptLine`.
    *
-   * @param template The template that will be cloned to create the
-   *     transcript line.
+   * @param timestampRange The timestamp for the transcript line.
    * @param transcriptLine The transcriptLine from `ENDPOINT_TRANSCRIPT`
    *     whose `attributes` should be used.
    */
-  constructor(template, transcriptLine) {
+  constructor(timestampRange, transcriptLine) {
     super();
-    this.attachShadow({mode: 'open'});
-    this.shadowRoot.appendChild(template.content.cloneNode(true));
+    const contentDivElement = TranscriptLineElement.createContentDivElement();
+    TranscriptLineElement.appendParagraphToContainer(
+        timestampRange, contentDivElement, ['justify-content-start', 'mb-1']);
+    TranscriptLineElement.appendParagraphToContainer(
+        transcriptLine.content, contentDivElement, ['ml-4', 'mb-1']);
+    this.classList.add('align-self-center', 'mb-2');
+    this.appendChild(contentDivElement);
+    this.appendChild(TranscriptLineElement.createHrElement());
```

Comment:
These function calls set up the element in the DOM by appending the various elements to it.